admin/src/components/footer.js

```diff
@@ -23,13 +23,8 @@ export default () => {
             </a>
           </li>
           <li>
-            <a href="https://www.snu.gouv.fr/nous-contacter-35" target="_blank">
-              Nous contacter
-            </a>
-          </li>
-          <li>
-            <a href="https://www.snu.gouv.fr/foire-aux-questions-11" target="_blank">
-              FAQ
+            <a href="https://admin.snu.gouv.fr/besoin-d-aide" target="_blank">
```

Comment:
c'est pas tres grave mais ducoup il y a un lien vers la prod en staging/local

je ferais plutot ca :
```suggestion
            <a href={`${ADMIN_URL}/besoin-d-aide`} target="_blank">
```

On ne devrait pas plutot pousser la base de connaissance ?